### PR TITLE
QF-2112-RefreshStandards_job_added

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: src/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests.csproj [hunk 1]
```diff
@@ -19,6 +19,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.169" />
+    <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests.csproj:22]: Why do we need this now?

File: src/SFA.DAS.RequestApprenticeTraining.Jobs/SFA.DAS.RequestApprenticeTraining.Jobs.csproj [hunk 2]
```diff
@@ -25,6 +25,8 @@
     <PackageReference Include="NLog" Version="5.3.2" />
     <PackageReference Include="NLog.Extensions.Logging" Version="5.3.11" />
     <PackageReference Include="SFA.DAS.NLog.Targets.Redis" Version="1.2.1" />
+    <PackageReference Include="System.Drawing.Common" Version="8.0.8" />
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Jobs/SFA.DAS.RequestApprenticeTraining.Jobs.csproj:28]: And these we are doing pretty much what we were doing before - odd that these are now required?

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests.csproj (hunk 1 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests/SFA.DAS.RequestApprenticeTraining.Jobs.UnitTests.csproj:22]: There were sonar cloud warning so i updated some of the packages 
image

File: src/SFA.DAS.RequestApprenticeTraining.Jobs/SFA.DAS.RequestApprenticeTraining.Jobs.csproj (hunk 2 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Jobs/SFA.DAS.RequestApprenticeTraining.Jobs.csproj:28]: Same as above

**Review by ChrisJWoodcock — APPROVED** [review]

**Review by furqanagwan — APPROVED** [review]

**Pull request merged**